src/cl_main.c

```diff
@@ -2114,7 +2114,7 @@ static double CL_MinFrameTime (void)
 		return 0;
 
 	if (cls.state == ca_disconnected || (Minimized && !cls.download))
-		return 1 / 30.0;
+		return 1 / r_displayRefresh.value;
```

Comment:
Or you could remove `cls.state == ca_disconnected` and that would be better since with the change you did you also affected FPS for a minimized situation and it would be better to not hog CPU in such a situation.

Reply:
so @zigg1zagg1 , are you going to update this PR based on qqshka's reply?